src/libraries/System.Private.CoreLib/src/System/GCMemoryInfo.cs

```diff
@@ -2,19 +2,81 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.InteropServices;
+
 namespace System
 {
+    // !!!!!!!!!!!!!!!!!!!!!!!
+    // make sure you change the def in vm\comutilnative.h
+    // if you change this!
+    public readonly struct GCGenerationInfo
```

Comment:
This should get triple-slash doc comment as well to get auto-generated documentation

Reply:
ahh right, will do.